core/cautils/scaninfo.go

```diff
@@ -188,25 +187,39 @@ func (scanInfo *ScanInfo) setUseFrom() {
 	}
 }
 
-func (scanInfo *ScanInfo) setOutputFile() {
-	if scanInfo.Output == "" {
-		return
-	}
-	if scanInfo.Format == "json" {
-		if filepath.Ext(scanInfo.Output) != ".json" {
-			scanInfo.Output += ".json"
+func (scanInfo *ScanInfo) GetFormats() []string {
```

Comment:
```suggestion
func (scanInfo *ScanInfo) getFormats() []string {
```
Is there a reason for this function to be public?

Reply:
`GetFormats` is defined in `cautils` package and called in both `cautils` and `core` packages.